torchdrivesim/rendering/base.py

```diff
@@ -42,6 +40,35 @@ class DummyRendererConfig(RendererConfig):
     backend: str = 'dummy'
 
 
+class Cameras:
+    """
+    Lightweight version of pytorch3d.renderer.FoVOrthographicCameras.
+    Used to define an interface that works without pytorch3d.
+    """
+    def __init__(self, xy: Tensor, sc: Tensor, scale: float):
+        self.xy = xy
```

Comment:
Should we enforce dimension for passed tensors and also keep track of the device used for this class ?

Reply:
I think I would prefer to keep it lightweight for now. I was thinking we might need to add some coherent shape checks throughout the library, and if we do, we could address this at the same time.